<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="TG5CD2141TFJ" failures="1" tests="1" name="[Nombre Proyecto]" time="59.149" errors="0" timestamp="2023-08-30T16:29:57 CLT">
  <testcase classname="test.java.main.PostulacionSocioTests" name="obtenerSegundoResultado" time="52.983">
    <failure type="java.lang.AssertionError" message="null">
        <![CDATA[java.lang.AssertionError: null
at main.java.config.Utils.eventFailed(Utils.java:109)
at main.java.utils.Events.getElementList(Events.java:122)
at main.java.pageEvents.ResultsEvents.resultVerify(ResultsEvents.java:24)
at main.java.pageSteps.ResultsSteps.getResult(ResultsSteps.java:12)
at test.java.main.PostulacionSocioTests.obtenerSegundoResultado(GoogleTests.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- obtenerSegundoResultado -->
</testsuite> <!-- [Nombre Proyecto] -->
